README.md

```diff
@@ -617,10 +617,6 @@ In C, functions are supposed to be idempotent, (without side effects). I hope th
 
 Within the body of a loop, assume that the loop action is successful and immediately update all pointer variables. If an exception is later detected on that loop action, back out the pointer advancements as side effects of a conditional expression following the loop body.
 
-#### Local Variables
-
-Within the body of a loop, assume that the loop action is successful and immediately update all pointer variables. If an exception is later detected on that loop action, back out the pointer advancements as side effects of a conditional expression following the loop body.
-
```

Comment:
It seems like I may have accidentally removed this section while doing formatting updates or something.

> Never use local variables. Whenever you feel the temptation to use one, make it into an instance or static variable instead to unselfishly share it with all the other methods of the class. This will save you work later when other methods need similar declarations. C++ programmers can go a step further by making all variables global.

http://mindprod.com/jgloss/unmaindesign.html under the section "Local Variables".